examples/main.ml

```diff
@@ -1,5 +1,10 @@
-Doggo.eleven_out_of_ten_majestic_af ~pupper:{
-  many=2112;
-  wow='x';
-  breed=C_Labrador
-}
+(* open Doggo *)
+
+module Doggo = struct
+  include Doggo_sys
+  (** Here is where we would write our wrappers around the raw bindings *)
+
+  let wrapper_print_age (age : int) = print_age ~age:(create_ptr age)
```

Comment:
This doesn't actually set the value yet... trying to think how to make that ergonomic, ideally without going through *another* FFI call.